src/scripts/main.js

```diff
@@ -1,3 +1,257 @@
+/* eslint-disable max-len */
```

Comment:
Remove the 'eslint-disable max-len' line, and ensure that your code does not exceed the maximum line length specified in the ESLint configuration.